### ISSUE TITLE
Espaço para ideias de novos projetos

### ISSUE THREAD

**Issue by danielrcoura** [OpenDevUFCG/IssueAi#59]
Talvez pudesse haver um espaço para sugestões de projetos.

Justificativa: às vezes temos boas ideias de projetos, infelizmente não temos tempo para desenvolver, mas queremos muito que exista.

Podem surgir boas ideias e pessoas interessadas em desenvolver :)

**Comment by fanny** [comment]
Muito bacana tua ideia, Daniel!

Atualmente usamos issues pra fomentar essas discussões, mas pretendemos criar sim uma estrutura pra isso.

cc @thayannevls @JoseRenan @SpinnelSun @JuanBarros2

**Comment by thayannevls** [comment]
Sim, podemos começar simples com um arquivo de ideias que permita contribuição e depois um espaço no Portal para isso

**Comment by danielrcoura** [comment]
Legal gente!

@fanny, onde eu posso abrir uma issue dessas?

**Comment by fanny** [comment]
Aqui mesmo, Danyiel.
